surpriseImpl/svd.py

```diff
@@ -0,0 +1,53 @@
+from surprise import SVD
+from surprise import Dataset
+from surprise.model_selection import cross_validate
+from surprise import accuracy
+from surprise.model_selection import train_test_split
+from dataloaders import movielens_data as loader
+from surprise import Reader
+
+
+def built_in_100k():
+    # Load the movielens-100k data set (download it if needed),
+    data = Dataset.load_builtin('ml-100k')
```

Comment:
loader den her det ud som samme format som vores egen dataloader?

Reply:
næh, vores loader er vel en pandas dataframe, det tror jeg ikke deres built-in er